modules/kairos-k3s-cluster/variables.tf

```
@@ -10,6 +10,15 @@ variable "additional_disks" {
   default = []
 }
 
+variable "additional_manifests" {
+  type = list(object({
+    content     = string
+    dest        = string
+    permissions = number
+  }))
+  default = []
+}
+
```

```suggestion
variable "additional_manifests" {
  type = list(object({
    name        = string
    content     = string
    permissions = optional(number)
  }))
  default = []
  description = "additional manifests for k3s to deploy on startup"
}
```
I would suggest we make permissions optional, since there is a sensible default, and that we automatically place additional manifests in the k3s manifests dir, since the scope of this var should probably just be for manifests rather than all files. Have made suggestions in `locals.tf` and `server.tf` to match